package.json

```diff
@@ -37,6 +37,7 @@
   "license": "LGPL 3",
   "scripts": {
     "build": "browserify -d ./src/main.js -p [minifyify --map peaks.min.map --output peaks.min.map] -s peaks -o peaks.min.js",
+    "build-max": "browserify ./src/main.js -s peaks -o peaks.js",
```

Comment:
I take the outputted `peaks.js` file and drop it into a Broccoli pipeline which minifies it and inlines it with my other JavaScript. It does not work very well when it tries to minify the already minified version.